telethon/events/newmessage.pyi

```diff
@@ -0,0 +1,6 @@
+from telethon.events.common import EventCommon
+from telethon.tl.custom import Message
+
+
+class NewMessage:
+    class Event(EventCommon, Message): ...
```

Comment:
I presume this can't be done directly in the `.py` file?

Reply:
No, because if it were included in the main py file, then the actual properties of the `Message` class would take over the proxy functioning.